src/graphql/types/Query/organizations.ts

```
@@ -0,0 +1,150 @@
+import {
+	QueryOrganizationInput,
+	queryOrganizationInputSchema,
+} from "~/src/graphql/inputs/QueryOrganizationInput";
+import { Organization } from "~/src/graphql/types/Organization/Organization";
+import { TalawaGraphQLError } from "~/src/utilities/TalawaGraphQLError";
+
+// Define type for organization model
+type OrganizationType = InferSelectModel<typeof organizationsTable>;
+export const organizationsTableQueryResolver = async (
+	_parent: unknown,
+	args: { input?: any },
```

Biome (1.9.4)

[error] Unexpected any. Specify a different type.

any disables many type checking rules. Its use should be avoided.

(lint/suspicious/noExplicitAny)